src/coreclr/tools/Common/Internal/Runtime/ReadyToRunConstants.cs

```diff
@@ -352,6 +352,8 @@ public enum ReadyToRunHelper
         MonitorEnterStatic,
         MonitorExitStatic,
 
+        NewMultiDimArr1,
```

Comment:
Adding this helper here would require R2R version bump.

Reply:
It comes after FirstFakeHelper - where do those show up in the file format?

Reply:
I have missed that. Ignore this comment.